docs/Guides/Ecosystem.md

```diff
@@ -252,6 +252,8 @@ section.
   plugin to authenticate HTTP requests based on API key and signature
 - [`fastify-appwrite`](https://github.com/Dev-Manny/fastify-appwrite) Fastify
   Plugin for interacting with Appwrite server.
+- [`fastify-asyncforge`](https://github.com/mcollina/fastify-asyncforge) Plugin 
+  to access Fastify instance, logger, request and reply from Node.js ASL.
```

Comment:
Can you expand the acronym and add a link?

Reply:
I think he means [AsyncLocalStorage](https://nodejs.org/api/async_context.html#class-asynclocalstorage). ALS instead of ASL??

Reply:
Absolutely, sorry for the typo. Updated on latest commit